packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -82,8 +81,16 @@ const parseFlags = (fetchedFlags: TFetchedFlags): TParsedHttpAdapterFlags =>
     ])
   );
 
+type TAdditionalEnvironmentProperties = {
+  env: {
+    enableFeatureConfigurationFetching?: boolean;
+  }
+};
 export const SetupFlopFlipProvider = (props: Props) => {
   const apolloClient = useApolloClient();
+  const enableFeatureConfigurationFetching = useApplicationContext<TAdditionalEnvironmentProperties>(
+    (context) => context.environment.env.enableFeatureConfigurationFetching
+  );
```

Comment:
@emmenko this is what you had in mind, right?

Reply:
Yes thanks